server/util/addTokenListsToDatabase.ts

```diff
@@ -31,17 +31,16 @@ async function addTokenListsToDatabase(models) {
 
   return Promise.all(tokens.map(async (token) => {
     // see if it already has it
-    const existing = await models.Token.findOne({ where: { address: token.address } });
-    if (!existing) {
-      return models.Token.create({
+    return models.Token.findOrCreate({
```

Comment:
Note that findOrCreate returns a different result than you expect: `[ modelInstance, isCreated ]` -- did you QA this?

Reply:
I thought i tested it! Let me check again

Reply:
I'm not sure it needs to be changed. The return value is only right now being checked for null/not-null

Reply:
In that case -- it should return an error code 0/numeric if success/failed, to be clear that we don't look at the result values.

Reply:
ok done